app/web/features/dashboard/CommunitiesList.tsx

```diff
@@ -49,21 +50,21 @@ export default function CommunitiesList({ all = false }: { all?: boolean }) {
             {communities.data.pages
               .flatMap((page) => page.communitiesList)
               .map((community) => (
-                <Link
+                <NextLink
```

Comment:
Is the `<StyledLink>` component suitable for what you want to do here?

Reply:
Oh yes! Thanks!